Homework 12. Sections What clients say and Meet our team.

VioletaScvortova opened the pull request:
Hello! I've added below 2 screenshots with my homework. Sections What clients say and Meet our team. 
There was some kind of error and the same commit "Added What Clients Say content" was created twice.
Looking forward to your reviews))

![Lesson 12_ Homework  Page What clients say](https://github.com/VioletaScvortova/Frontend-Project-KMarketer/assets/141768669/dc59123a-867b-4ad8-8546-4b374a0d8582)
![Lesson 12_ Homework  Page Meet our team](https://github.com/VioletaScvortova/Frontend-Project-KMarketer/assets/141768669/9b4ec6c9-ff25-4bc6-b29b-1c6c2e2f5896)

Irrreny commented on index.html:
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+            <div class="grid-container">
+                <div class="grid-card" id="card-items">
+                    <div class="stars">
+                        <span data-rate="1"></span>
```
I think you could use simple images here and add only width and height in css, so in this way you could have saved some code. But i guess it works both ways :)

VioletaScvortova replied:
I`ve added url with images (svg) to .stars and .empty-stars classes. I found this way to display the rating and I also have a question if it`s ok to use this way and to bother with it))

Thank you a lot for your review!

Irrreny commented on index.html:
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+                        <div>
+                            <div class="staff__content">
+                                <h5 class="staff__name">Ashley Fletcher</h5>
+                                <h6 class="staff__job">CEO</h6>
```
You can style capital letters in css.

VioletaScvortova replied:
Oh, yees, exactly, thank you! I forgot about the possibility - "text-transform: uppercase;"

LiluUncharted commented on index.html:
```diff
@@ -66,10 +66,11 @@ <h4 class="content__mission">We know how large objects will act, but things on a
+
             <div class="strong-points__content">
                 <h2 class="content__title">Why choose us</h2>
                 <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
```
"br" not the best option to use, it's better to set styles in css, just assign a max-width and align to the center )

VioletaScvortova replied:
I'll keep this in mind for the future, thank you for your advice!

IvanCeban commented on index.html:
```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+                    </div>
+                </div>
+
+                <div class="grid-card" id="card-staff">
```
id should be unique.
You use `id="card-staff"` several times, which is forbidden

IvanCeban replied:
it's not only about card-staff. Check all your ids

VioletaScvortova replied:
done!

VioletaScvortova replied:
@IvanCeban thanks a lot for the detailed explanation! Now it`s clear

IvanCeban reviewed (changes requested):
Very well done!
Please check the comments

VioletaScvortova commented:
@IvanCeban hello! I've changed the rating format and added a section with the company video presentation. Please, check if everything is well done. 

As for the gradient using png, I searched on google and double-checked a lot, but I still couldn’t find the reason why this option didn’t work.

![Lesson 12_ Homework  Page Video presentation](https://github.com/VioletaScvortova/Frontend-Project-KMarketer/assets/141768669/29f59f0f-2a31-40bb-9c8c-269bef2e1fa4)
